### PR TITLE
Add user as json to cookie on signin instead of only user id

### PR DESCRIPTION
**Pull request by anttilip**
Since `/authenticate` return whole user object, it should be stored to `user` cookie. This reduces future fetches.

### CODE REVIEW

File: evento/src/views/MyEvents/index.js [hunk 1]
```diff
@@ -17,16 +17,18 @@ class MyEvents extends Component {
 	}
 	
 	fetchEvents() {
-		const userId = Cookie.get('userId');
+		let user = Cookie.get('user');
```
Comment (JaakkoLipsanen) [inline evento/src/views/MyEvents/index.js]: Yeah no. We aren't gonna keep both the JSON presentation and the parsed value in the same variable. That is just a bad practice and confusing. Rename this to `const userCookie` or something

**Comment by anttilip** [comment]
>Fix the one issue, otherwise good.

Which issue? I already renamed `user` to `userCookie` in 668d423.

**Comment by JaakkoLipsanen** [comment]
Oh okay, didn't see it. Ready to merge?

**Comment by anttilip** [comment]
> Ready to merge?

Yep!

**Pull request merged**